content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -9,13 +9,15 @@ weight: 65
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.27" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.30" state="alpha" >}}
 
 Dynamic resource allocation is an API for requesting and sharing resources
 between pods and containers inside a pod. It is a generalization of the
 persistent volumes API for generic resources. Third-party resource drivers are
-responsible for tracking and allocating resources. Different kinds of
-resources support arbitrary parameters for defining requirements and
+responsible for tracking and allocating resources, with additional support
+provided by Kubernetes via "structured parameters" (introduced in Kubernetes 1.30).
+Different kinds of resources support arbitrary parameters for defining requirements and
```

Comment:
I think we should expand on the distinction here a bit more in the intro.

Reply:
I added one sentence:
> When a driver uses structured parameters, Kubernetes handles scheduling
> and resource allocation without having to communicate with the driver.